librosa/sequence.py

```diff
@@ -397,9 +416,14 @@ def __dtw_backtracking(D_steps, step_sizes_sigma, subseq):  # pragma: no cover
     --------
     dtw
     '''
+    if start_m is None:
+        cur_idx = (D_steps.shape[0] - 1, D_steps.shape[1] - 1)
+    else:
+        assert subseq
```

Comment:
assertions should only be used in tests, not library code.  If there's something wrong with the input value, raise an exception with an informative error message.